tests/conftest.py

```
@@ -14,6 +15,7 @@ def config_file() -> str:
 
 @pytest.fixture(scope="session", autouse=True)
 def setup(config_file: str):
-    tests.setup(config_file)
-    yield
-    tests.teardown()
+    with mock.patch("acoustid.data.fingerprint.SEARCH_ONLY_IN_DATABASE", True):
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)